regen-network/regen-web
# feat: Add requested metadata to buy, send, and sell events

## Description

Closes: https://github.com/regen-network/regen-registry/issues/1341



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. Go to the deploy preview: https://deploy-preview-1551--regen-registry.netlify.app/
2. Open the "Regen Marketplace [DEV]" GA dashboard (realtime tab): https://analytics.google.com/analytics/web/#/p340668480/realtime/overview?params=_u..nav%3Dmaui
3. Complete all actions in the tracking spreadsheet: https://docs.google.com/spreadsheets/d/1JXw_pDC4QyqtcdYgxpMkbTVc24E2yz-qpwxvzelPfnY/edit?usp=sharing
    a. At each step, check that you see the corresponding event in GA or Amplitude.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
